paparazzi/src/main/java/app/cash/paparazzi/Paparazzi.kt

```diff
@@ -165,6 +177,22 @@ class Paparazzi @JvmOverloads constructor(
 
   fun <V : View> inflate(@LayoutRes layoutId: Int): V = layoutInflater.inflate(layoutId, null) as V
 
+  @JvmOverloads
+  fun snapshot(
+    composable: @Composable () -> Unit,
+    name: String? = null,
+    deviceConfig: DeviceConfig? = null,
+    theme: String? = null,
+    renderingMode: RenderingMode? = null
+  ) {
+    val hostView = ComposeView(context)
+    val parent = FrameLayout(context).apply { id = android.R.id.content }
```

Comment:
Thoughts on adding a comment on why this ID is needed for our future selves? This is such a hidden implementation detail.

Reply:
hidden and impressive*